content/04.workflows-and-software.md

```diff
@@ -67,29 +67,31 @@ The [Getting started developing workflows](### getting-started-developing-workfl
 ### Wrangling Scientific Software
 
 Analysis workflows commonly rely on multiple software packages to generate final results.
-These tools are heterogeneous in nature: written by researchers working in different coding languages, with varied approaches to software design and optimization, and often for specific analysis goals.
+These tools are heterogeneous in nature: they are written by researchers working in different coding languages, with varied approaches to software design and optimization, and often for specific analysis goals.
 Each program has a number of other programs it depends upon to function ("dependencies"), and as software changes over time to meet research needs, the results may change, even when run with identical parameters.
 As a result, it is critical to take an organized approach to installing, managing, and keeping track of software and software versions.
-To meet this need, most workflow managers integrate with software management systems like conda, singularity, and docker [@doi:10.1038/s41592-018-0046-7; @doi:10.1371/journal.pone.0177459; @doi:10.5555/2600239.2600241].
+To meet this need, most workflow managers integrate with software management systems like conda, singularity, and docker [@doi:10.1038/s41592-018-0046-7; @doi:10.1371/journal.pone.0177459; @doi:10.5555/2600239.2600241]. (CTB: one of these citations fails.)
```

Comment:
it's the docker citation (though it seems to be real) https://dl.acm.org/doi/10.5555/2600239.2600241